packages/client/components/transactions-list/SortTransactionListComponent.tsx

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+import { Actionsheet, useDisclose, Radio, Text, HStack, Pressable } from "native-base";
+import { StyleSheet } from "react-native";
+import { faSort } from "../icons/regular/farSort";
+import { FontAwesomeIcon } from "@fortawesome/react-native-fontawesome";
+import { View } from "../Themed";
+
+type Props = {
+    sortType: string;
+    setSortType: React.Dispatch<React.SetStateAction<string>>;
+};
+
+function SortTransactionListComponent({ sortType, setSortType }: Props) {
+    const { isOpen, onOpen, onClose } = useDisclose();
+
+    const onPressReset = () => {
+        setSortType("sortDateNewest");
+    };
+
+    return (
+        <>
+            <View>
+                <Pressable onPress={onOpen} style={{ paddingRight: 4.5 }}>
+                    <FontAwesomeIcon icon={faSort} size={19} />
+                </Pressable>
+
+                <Actionsheet isOpen={isOpen} onClose={onClose}>
+                    <Actionsheet.Content>
+                        <HStack style={styles.headerStack}>
+                            <Text style={styles.resetStyleInvisible}>Reset</Text>
+
+                            <Text fontWeight={"semibold"} style={styles.headerStyle}>
+                                Sort
+                            </Text>
+
+                            <Text
+                                onPress={onPressReset}
```

Comment:
You can inline this function, no point to extracting it

```suggestion
                                onPress={() => setSortType("sortDateNewest")}
```